packages/strapi-tiptap-editor/admin/src/components/extensions/Price/index.ts

```diff
@@ -30,27 +30,29 @@ export const Price = Node.create({
   content: 'inline*',
   inline: true,
   draggable: true,
-  atom: true,
   addAttributes() {
     return {
-      id: {
+      'data-strapi-idref': {
         default: '',
       },
+      'data-strapi-category': {
+        default: 'price',
+      },
     };
   },
   addCommands() {
     return {
       insertReactComponent:
         (price) =>
         ({ commands }) => {
-          return commands.insertContent(`<react-widget id='${price.id}'></react-widget>`);
+          return commands.insertContent(`<span data-strapi-idref='${price.id}'></span>`);
```

Comment:
Mist hier `data-strapi-category`?

Reply:
Het `data-strapi-category` attribuut is niet aanpasbaar, dus het is niet nodig om het hier opnieuw toe te voegen.